src/test/java/seedu/duke/helper/ParserTest.java

```diff
@@ -25,7 +25,7 @@ void printDetails(TestInfo testInfo) {
     void validateMedicineTest() {
         String[] userInputArray = {"paracetamol", "500", "2023-02-02", "Headaches", "500"};
         String userInput = "paracetamol,500,2023-02-02,Headaches,500";
-        assertEquals(true, Parser.validateMedicine(userInputArray));
+        assertEquals(true, Validator.validateMedicine(userInputArray));
```

Comment:
I think there is a issue with this JUnit Test, you will have to create a new test class since validator is a separate class.